tests/appsec/iast_packages/test_packages.py

```diff
@@ -120,45 +207,216 @@ def install_latest(self):
+    ),
+    PackageForTesting(
+        "s3transfer",
+        "0.10.1",
+        "",
+        "",
+        "",
+        extras=[("boto3", "1.34.110")],
+    ),
+    # TODO: Test import fails with
+    #   AttributeError: partially initialized module 'setuptools' has no
+    #   attribute 'dist' (most likely due to a circular import)
+    PackageForTesting(
+        "setuptools",
+        "70.0.0",
+        "",
+        {"description": "An example package", "name": "example_package"},
+        "",
+        test_import=False,
     ),
-    PackageForTesting("requests", "2.31.0", "", "", ""),
+    PackageForTesting("six", "1.16.0", "", "We're in Python 3", ""),
+    # TODO: Test import fails with AttributeError: 'FormattedValue' object has no attribute 'values'
+    # PackageForTesting("soupsieve", "2.5", "", "", "", test_e2e=False,
+    # import_module_to_validate="soupsieve.css_match"),
+    PackageForTesting("tomli", "2.0.1", "", "", "", test_e2e=False, import_module_to_validate="tomli._parser"),
+    PackageForTesting("tomlkit", "0.12.5", "", "", "", test_e2e=False, import_module_to_validate="tomlkit.items"),
+    PackageForTesting("tqdm", "4.66.4", "", "", "", test_e2e=False, import_module_to_validate="tqdm.std"),
+    # Python 3.8 and 3.9 fial with ImportError: cannot import name 'get_host' from 'urllib3.util.url'
     PackageForTesting(
         "urllib3",
         "2.1.0",
         "https://www.datadoghq.com/",
         ["https", None, "www.datadoghq.com", None, "/", None, None],
         "www.datadoghq.com",
+        skip_python_version=[(3, 8), (3, 9)],
     ),
-    PackageForTesting("beautifulsoup4", "4.12.3", "<html></html>", "", "", import_name="bs4"),
-    PackageForTesting("setuptools", "70.0.0", "", "", "", test_e2e=False),
-    PackageForTesting("six", "1.16.0", "", "", "", test_e2e=False),
-    PackageForTesting("s3transfer", "0.10.1", "", "", "", test_e2e=False),
-    PackageForTesting("certifi", "2024.2.2", "", "", "", test_e2e=False),
-    PackageForTesting("cryptography", "42.0.7", "", "", "", test_e2e=False),
-    PackageForTesting("fsspec", "2024.5.0", "", "", "", test_e2e=False, test_import=False),
-    PackageForTesting("boto3", "1.34.110", "", "", "", test_e2e=False, test_import=False),
-    # Python 3.8 fails in test_packages_patched_import with
-    # TypeError: '>' not supported between instances of 'int' and 'object'
-    # TODO: try to fix it
+    PackageForTesting(
+        "virtualenv", "20.26.2", "", "", "", test_e2e=False, import_module_to_validate="virtualenv.activation.activator"
+    ),
+    # TODO: Test import fails with AttributeError: 'FormattedValue' object has no attribute 'values'
+    # PackageForTesting("werkzeug", "3.0.3", "", "", "", test_e2e=False, import_module_to_validate="werkzeug.http"),
+    PackageForTesting("yarl", "1.9.4", "", "", "", test_e2e=False, import_module_to_validate="yarl._url"),
+    PackageForTesting("zipp", "3.18.2", "", "", "", test_e2e=False),
+    # PENDING TO TEST
```

Comment:
Not pending to test, it's definitely not us but an issue in AST module in Python 3.8. It also fails with an empty AST visitor.